bin/install-qa-check.d/60pkgconfig

```diff
@@ -102,23 +102,35 @@ pkgconfig_check() {
 	if [[ -n ${QA_PKGCONFIG_VERSION} ]]; then
 		local all_bad=yes
 		# Record the last bad file matched
-		local bad_file
+		local -A bad_files
 		for f in "${files[@]}" ; do
-			if [[ ${QA_PKGCONFIG_VERSION} == $(pkg-config --modversion "${f}") ]] ; then
+			local file_version=$(pkg-config --modversion "${f}")
+			if [[ ${QA_PKGCONFIG_VERSION} == ${file_version} ]] ; then
```

Comment:
Could this comparison be aware of version components, so that e.g. it treats `25.0` and `25.0.0` as equal?

Reply:
Indeed, it believe using `ver_test ${QA_PKGCONFIG_VERSION} -eq ${file_version}` would achieve this. I am not sure if `ver_test` is available in `install-qa-check.d`, will have a look.